src/App.js

```diff
@@ -1,26 +1,127 @@
 import React from 'react';
 import logo from './logo.svg';
+import foods from './foods.json';
+import FoodBox from './components/FoodBox';
+import AddFood from './components/AddFood';
+import DayFood from './components/DayFood';
+import 'bulma/css/bulma.css';
 import './App.css';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends React.Component {
+  state = {
+    allFood: foods,
+    filteredFood: foods,
+    todayFood: [],
+    showForm: false,
+    calories: 0
+  };
+
+  
+
+  handleSearch = (event) => {
+    const search = event.target.value;
+    const { allFood } = this.state;
+
+    let filteredFood = allFood.filter((food) => {
+      return food.name.toLowerCase().includes(search.toLowerCase());
+    });
+
+    this.setState({
+      filteredFood: filteredFood,
+    });
```

Comment:
to reduce some code you can put the setState in a single line, also because the key and variable hold the same name, you can use a single "filteredFood". Like below:
`this.setState({ filteredFood });`